config/locales/en.yml

```diff
@@ -3,6 +3,9 @@ en:
   hub:
     unlock_account: "Unlock account"
     unlock_confirmation: "Unlock %{name}'s account? Have they confirmed their identity?"
+    suspend_account: "Suspend account"
+    suspend_confirmation: "Suspend %{name}'s account?"
```

Comment:
"Suspend %{name}'s account? This will remove them from all currently assigned tax returns and prevent them from logging in." Might help 'self-document' what this feature does.